addons/wp/stringtable.xml

```diff
@@ -172,6 +172,12 @@
         <Key ID="STR_Lambs_WP_Module_TaskCreep_MovingCenter_ToolTip">
             <English>The creeping group will use the group leader as a center for the search pattern. Disable to have the unit use the module position instead. This creates a more defensive group</English>
         </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCreep_PlayersOnly_DisplayName">
+            <English>Players Only</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCreep_PlayersOnly_ToolTip">
+            <English>The creeping group will only Use Players as targets.</English>
+        </Key>
```

Comment:
Why not just one translation for this? Means we have for the none tool tip duplicates. Might as well add placeholders and for `format` on the translations

Reply:
I wanted to keep it the same with all the other modules and later have a pass to remove all duplicates.